### PR TITLE
(port) Android Sect can now trade favor for cybernetic implants

### PR DESCRIPTION
**Pull request by boodaliboo**
original pr : https://github.com/tgstation/tgstation/pull/60024
"you may now use 2000 favor to get a set of robotic implants as the chaplain, i've tried to make them as non power gamey as possible in a way where they are useful but not necessarily an advantage over people"
implants avaiable
thrusters
med hud
breathing tubes
flashlight eyes
diagnostic arms
surgery arms
:cl:  
rscadd: adds a new rite to android religion for 2000 favor create a implant 
/:cl:

### CODE REVIEW

**Comment by Moggram** [comment]
bro.... 2000? that seems a bit high. it takes like 45 or 48 blue space cells just for the synth conversion.

oh shit, and its just for one?!

**Comment by maxion12345** [comment]
diagnostic arms? the fuck are those

also it does seem a bit of a big cost for an implant

**Comment by boodaliboo** [comment]
> diagnostic arms? the fuck are those
> 
> also it does seem a bit of a big cost for an implant

whats a better ammount?

**Review by SomeguyManperson — APPROVED** [review]

File: code/modules/religion/rites.dm [hunk 1]
```diff
@@ -102,6 +102,29 @@
...
 	return TRUE
 
+/datum/religion_rites/machine_blessing
+	name = "Receive Blessing"
+	desc = "Receive a blessing from the machine god to further your ascension."
+	ritual_length = 5 SECONDS
+	ritual_invocations =list( "Let your will power our forges.",
+							"...Help us in our great conquest!")
+	invoke_msg = "The end of flesh is near!"
+	favor_cost = 2000
```
Comment (SomeguyManperson) [inline code/modules/religion/rites.dm]:
```suggestion
	favor_cost = 200	
```

File: code/modules/religion/religion_sects.dm [hunk 2]
```diff
@@ -112,8 +112,7 @@
 	convert_opener = "May you find peace in a metal shell, acolyte.<br>Bibles now recharge cyborgs and heal robotic limbs if targeted, but they do not heal organic limbs. You can now sacrifice cells, with favor depending on their charge."
 	alignment = ALIGNMENT_NEUT
 	desired_items = list(/obj/item/stock_parts/cell)
-	rites_list = list(/datum/religion_rites/synthconversion,
-					  /datum/religion_rites/botcreation)
+	rites_list = list(/datum/religion_rites/synthconversion,/datum/religion_rites/botcreation, /datum/religion_rites/machine_blessing)
```
Comment (JamieD1) [inline code/modules/religion/religion_sects.dm]:
```suggestion
	rites_list = list(/datum/religion_rites/synthconversion, /datum/religion_rites/botcreation, /datum/religion_rites/machine_blessing)
```

**Pull request merged**